src/plots/histogram.rs

```diff
@@ -0,0 +1,440 @@
+use crate::error;
+use crate::plots::{Plot, PlotData};
+use crate::primitives::{DataRef, FeatureDataRef, Measurement, NullableDataRef};
+use crate::util::Result;
+use float_cmp::*;
+use ndarray::{stack, Array, Array1, Axis};
+use serde::{Deserialize, Serialize};
+use snafu::ensure;
+use std::collections::HashMap;
+use std::iter::FromIterator;
+use vega_lite_3::{
+    BinEnum, EncodingBuilder, Mark, Padding, SelectionDefBuilder, SelectionDefType,
+    SingleDefUnitChannel, StandardType, VegaliteBuilder, X2ClassBuilder, XClassBuilder,
+    YClassBuilder,
+};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct Histogram {
+    counts: Vec<u64>,
+    labels: Option<Vec<String>>,
+    nodata_count: u64,
+    min: f64,
+    max: f64,
+    measurement: Measurement,
```

Comment:
doesn't every plot measure something? should maybe reflected in Plot trait

Reply:
I doubt I can add a required field to the trait.
For a scatter plot, I have two Measurement-Types, for x and for y.

Reply:
we could have a getMeasuredValues() -> vec<Measurement> in the trait but we can add it when we have actual use for it

Reply:
I would prefer deferring it since I do not see the necessity at this moment.